backend/fleet_management/migrations/0016_auto_20190829_2155.py

```diff
@@ -0,0 +1,53 @@
+from django.db import migrations
+from django.contrib.auth.models import Group
+from faker import Faker
+
+from fleet_management.constants import Groups
+from ..models import User, Passenger, Drive
+
+
+def create_users_from_passengers(apps, schema_editor):
+    passenger_group = Group.objects.get(name=Groups.Passenger.name)
+
+    passengers = Passenger.objects.all()
+
+    for index, passenger in enumerate(passengers):
+        fake = Faker()
+        password = fake.password()
+
+        email = passenger.email or f"no-email{index}@pah.org.pl"
+
+        user = User.objects.create_user(
+            username=email,
+            email=email,
+            password=password,
+            is_superuser=False,
+            is_staff=False,
+            country=passenger.country,
+            is_active=True,
+            first_name=passenger.first_name,
+            last_name=passenger.last_name
+        )
+
+        user.save()
```

Comment:
That's not needed, see https://docs.djangoproject.com/en/2.2/ref/contrib/auth/#django.contrib.auth.models.UserManager.create_user